src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java

```diff
@@ -1148,6 +1149,36 @@ public AnalysisProvenanceResponse getProvenanceByFile(@PathVariable Long submiss
 		return analysisProvenance;
 	}
 
+	/**
+	 * Get the updated state and duration of an analysis
+	 *
+	 * @param submissionId The analysis submission id
+	 * @return dto which contains the updated analysis state and duration
+	 */
+	@RequestMapping(value = "/{submissionId}/updated-progress")
+	@ResponseBody
```

Comment:
Because this is a `RestController` you don't have to annotate methods with @responsebody.

Reply:
Removed in c64a856128d212fdda1bcab36dbf56b78cef8399